protobuf/protoc-gen-govalidator/plugin/plugin.go

```diff
@@ -243,6 +243,14 @@ func (p *plugin) generateLengthValidator(variableName string, ccTypeName string,
 func (p *plugin) generateInnerMessageValidator(variableName string, ccTypeName string, fieldName string, field *descriptor.FieldDescriptorProto, nullable bool) {
 
 	if nullable {
+		if isMsgRequired(field) {
+			p.P(`if `, variableName, ` == nil {`)
+			p.In()
+			p.P(`return fmt.Errorf("message `, fieldName, ` is required")`)
+			p.Out()
+			p.P(`}`)
+		}
```

Comment:
`else` ?

Reply:
I need both code blocks to be outputted.